### PR TITLE
Adding clipPath, filter and transform to waapi values

### PR DESCRIPTION
**Pull request by mattgperry**

### CODE REVIEW

File: packages/framer-motion/src/animation/utils/default-transitions.ts [hunk 1]
```diff
@@ -48,9 +38,11 @@ export const getDefaultTransition = (
 ): Partial<AnimationOptions> => {
     if (keyframes.length > 2) {
         return keyframesTransition
-    } else {
-        const factory =
-            defaultTransitions[valueKey] || defaultTransitions.default
-        return factory(keyframes[1])
+    } else if (transformProps.has(valueKey)) {
```
Comment (mattgperry) [inline packages/framer-motion/src/animation/utils/default-transitions.ts:41]: `opacity` `backgroundColor` and `color` now use this slightly smoother default easing curve

Comment (nvh) [reply packages/framer-motion/src/animation/utils/default-transitions.ts:41]: So is this a breaking change? (I think we can treat it as not?)

**Review by nvh — APPROVED** [review]

File: packages/framer-motion/src/animation/utils/default-transitions.ts (hunk 1 above)
Comment (mattgperry) [reply packages/framer-motion/src/animation/utils/default-transitions.ts:41]: Yeah technically this does change the defaults but personally I don't think anyone is going to notice this difference

**Review by hemlok — APPROVED** [review]
:heavy-breathing:

**Pull request merged**